skills/safe-operations/scripts/Add-FollowUpIssue.ps1

```
@@ -0,0 +1,109 @@
+    }
+
+    # 2. GraphQL Linkage with retry
+    # Get parent and child GraphQL Node IDs
+    $parentId = $null
+    $childId = $null
+    try {
+        $parentId = gh issue view $ParentIssue --json id --jq .id 2>$null
+        $childId = gh issue view $childNumber --json id --jq .id 2>$null
+    } catch {
+        # Fail silent, handled by the loop
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)